src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -270,8 +287,12 @@ private void configureDataPersistence() throws IOException {
                         .getProperty(Constants.REMOTE_GIT_SSH_KEY_PATH)));
 
         bind(IUserGroupPersistenceManager.class).to(PgUserGroupPersistenceManager.class);
-
         bind(IAssociationDataManager.class).to(PgAssociationDataManager.class);
+        bind(IAssignmentPersistenceManager.class).to(PgAssignmentPersistenceManager.class);
+        bind(IQuizAssignmentPersistenceManager.class).to(PgQuizAssignmentPersistenceManager.class);
+        bind(IQuizAttemptPersistenceManager.class).to(PgQuizAttemptPersistenceManager.class);
+        bind(IQuizQuestionAttemptPersistenceManager.class).to(PgQuizQuestionAttemptPersistenceManager.class);
+        bind(IUserBadgePersistenceManager.class).to(PgUserBadgePersistenceManager.class);
```

Comment:
These are just straight copied from the Isaac Guice config.

Reply:
`IUserBadgePersistenceManager` has been added, instead of the singleton provider which was overkill.